### DISCUSSION TITLE
there are a roadmap about use prisma with Go ?

### DISCUSSION THREAD

**Discussion by jcmartins** [prisma/prisma#6695] [Q&A] [answered]
I know about prisma-client-go but I didn't found any information about roadmap, how-to,  blog article or youtube videos

tks

**Comment by danielolaviobr** [comment, marked as answer]
Hey @jcmartins  There is a quick start example in the github repo, here you _Go_ 

https://github.com/prisma/prisma-client-go/blob/master/docs/quickstart.md

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
